src/public/modules/forms/admin/directives/settings-form.client.directive.js

```diff
@@ -144,11 +143,9 @@ function settingsFormDirective(
           )
         }
 
-        $scope.isDisableAuthType = (authType) => {
+        $scope.isDisable = () => {
```

Comment:
why rename this from `isDisableAuthType` to `isDisable`? actually this function is still specific to disabling the auth type right

Reply:
The reason I refactored this is because it originally took in an authType argument that is only used when checking for authType.val == sgid.
Currently it no longer checks for authType but instead mainly checks if the form is public or if there is an myInfoSP warning.

Also I tried to align with the React version which is in AuthSettingsSections.tsx and uses the method name `isDisabled`

P.S. I realised I should have named the method as `isDisabled` instead hahaha.